source/target/nordic/target_reset.c

```diff
@@ -45,24 +45,55 @@ uint8_t target_set_state(TARGET_RESET_STATE state)
 
 void swd_set_target_reset(uint8_t asserted)
 {
+    extern char *board_id;  //TODO - remove
```

Comment:
This TODO is also is also in info.c. I simply places the TODO to easier find it.
What was the reason behind the info.c TOTO remove board_id? Any plans there?